pages/chat/messages.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from "react";
+import { GiftedChat, IMessage } from 'react-native-gifted-chat';
+import firebase from 'firebase/compat/app';
+import 'firebase/compat/firestore';
+import { firestore, messageQuery, firebaseAuth } from "../../firebaseConfig";
+import {  addDoc, collection, query, orderBy, onSnapshot, DocumentData, Unsubscribe } from 'firebase/firestore';
+import TouristsNavbar from "../../custom_components/TouristsNavbar";
+
+
+export default function Chat() {
```

Comment:
I think we shld rename either this function or this file cos its a bit confusing for messages.ts to be exporting a func called Chat when there is another page called chat.tsx

Also might wanna consider changing the names chat and messages cos they're a bit too similar...maybe chatLanding and chat?